app/mailers/registration_mailer.rb

```diff
@@ -0,0 +1,16 @@
+class RegistrationMailer < ApplicationMailer
```

Comment:
You need some mailer specs for this: https://relishapp.com/rspec/rspec-rails/docs/mailer-specs

Reply:
Soo, I added a simple test for checking who email is sent to and from. 

I am pretty certain writing one for email body and subject is counter-productive, because I opted to use notify email templates.

It is possible to store emails in our codebase (in erb files), but templates let non-technical people edit copy whenever they please, so long as we invite them to our notify space. Happy to discuss trade-offs.

Reply:
Sounds good to me. We can always move to erb if it turns out to be a nightmare